R/lineprof.r

```diff
@@ -69,11 +72,15 @@
 #' }
 #' @useDynLib lineprof
 #' @importFrom Rcpp sourceCpp
-lineprof <- function(code, interval = 0.001, torture = FALSE) {
-  path <- line_profile(code, interval, torture)
-  on.exit(unlink(path))
+lineprof <- function(code, interval = 0.001, torture = FALSE, prof_path = "") {
 
-  parse_prof(path)
+  if (prof_path=="") {
```

Comment:
Can you please fix the spacing here? See the section on spacing at http://adv-r.had.co.nz/Style.html